src/components/banner/template.njk

```diff
@@ -1,18 +1,43 @@
-      <path d="M67.47,7.72a59.76,59.76,0,1,0,59.75,59.75A59.75,59.75,0,0,0,67.47,7.72m67.47,59.75A67.47,67.47,0,1,1,67.47,0a67.47,67.47,0,0,1,67.47,67.47" />
-    </svg>
+    {% if params.useTudorCrown | default(true) %}
+      <!--[if gt IE 8]><!-->
```

Comment:
Hi @oscarduignan , I see we've still got the IE8 fallback in here which is now removed from the `govuk-frontend` templates, shall we delete these fallbacks?

Reply:
done!